src/main/java/com/itspace/job/endpoint/UserEndPoint.java

```diff
@@ -31,5 +34,13 @@ public UserEndPoint(UserService userService, ModelMapper mapper) {
         }
         return ResponseEntity.status(400).build();
     }
+    @PostMapping("/login")
+    public  ResponseEntity<LoginResponseDto> login(@RequestBody LoginUser loginUser){
+        LoginResponseDto login = userService.login(loginUser);
+        if (login != null){
+            return ResponseEntity.ok(login);
+        }
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
```

Comment:
please change here HttpStatus.BAD_REQUEST status to HttpStatus.UNAUTHORIZED
The rest was done good.
thank you !!!